lib/routes/account.js

```diff
@@ -1477,6 +1478,8 @@ module.exports = (
               session.lastAccessTimeFormatted = localizeTimestamp.format(session.lastAccessTime,
                 request.headers['accept-language'])
 
+              session.os = session.uaOS || 'Unknown'
```

Comment:
Oh ok, I had a version like that, but also saw `desktop` and `mobile` were not a localized. I'll go with `null` 👍